core/utils/poly-look/src/visualisations/charts/line-charts/time-line-chart/timeLineChart.css

```diff
@@ -0,0 +1,7 @@
+.time-line-chart {
+  color: "#0f1938";
+}
+
+.time-line-chart * {
+  color: "#0f1938";
+}
```

Comment:
Nit: Why not do this in one rule?

```
.time-line-chart, .time-line-chart * {
    color: "#0f1938";
}
```

Also, isn't this one of our standard colours we have a CSS var for somewhere?